purview/models/distgit.py

```diff
@@ -0,0 +1,26 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+from neomodel import (
+    StructuredNode, UniqueIdProperty, RelationshipTo, StringProperty,
+    DateTimeProperty)
+
+
+class DistgitCommit(StructuredNode):
+    id_ = UniqueIdProperty(db_property='id')  # commit_id
+    owner = RelationshipTo('.user.User', 'OWNED_BY')  # author
+    committer = RelationshipTo('.user.User', 'OWNED_BY')
```

Comment:
Do you think adding these relationships to the User class as well and including that in this PR is a good idea?